config/initializers/ezid_client.rb

```diff
@@ -0,0 +1,6 @@
+Ezid::Client.configure do |config|
+  config.default_shoulder = "ark:/99999/fk4"
+  config.user = "apitest"
+  config.password = "apitest"
```

Comment:
My specific concern is that in the ezid-client documentation they instruct that the test user password is not provided and EZID should be contacted. We probably shouldn't be publishing, even in a config that gets overridden.